index.html

```diff
@@ -91,26 +91,40 @@ <h2>Stages</h2>
         <li>Initial spec text
       </ul>
     </td>
-    <td>The committee expects the feature to be developed and eventually included in the standard
+    <td>The committee expects the feature to be developed and eventually included in the standard. No changes to the proposal will be requested aside from those elicited through testing or implementation experience.
```

Comment:
this applies to the new stage, i think, but definitely does not apply to stage 2, in which entire proposals have changed, often.

Reply:
Oops, I think I edited the wrong line. Will fix.